dhall/src/Dhall/Util.hs

```diff
@@ -171,13 +168,13 @@ data Output = StandardOutput | OutputFile FilePath
 data OutputMode = Write | Check
 
 -- | Exception thrown when the @--check@ flag to a command-line subcommand fails
-data CheckFailed = CheckFailed { command :: Text, modified :: Text }
+data CheckFailed = CheckFailed { command :: Text, modified :: Text, input :: Input }
```

Comment:
Perhaps move the `command` and `modified` fields out of the `CheckFailed` type and into the `MultipleCheckFailed` type, since all the checks are going to share the same `command` / `modified` values.

This would also imply consolidating their error messages by moving the "You ran …" error header into `MultipleChecksFailed` so that it's only displayed once for all of the checks.